tests/test_2023_09_12.py

```diff
@@ -0,0 +1,93 @@
+"""
+Tests for voting 12/09/2023
+
+"""
+from scripts.vote_2023_09_12 import start_vote
+
+from utils.config import (
+    network_name,
+    contracts,
+    LDO_HOLDER_ADDRESS_FOR_TESTS,
+)
+from utils.voting import find_metadata_by_vote_id
+from utils.ipfs import get_lido_vote_cid_from_str
+from utils.test.tx_tracing_helpers import *
+
+from utils.test.event_validators.node_operators_registry import (
+    validate_target_validators_count_changed_event,
+    TargetValidatorsCountChanged,
+)
+from utils.test.event_validators.permission import validate_grant_role_event
+
+
+def test_vote(
+    helpers,
+    bypass_events_decoding,
+    vote_ids_from_env,
+    accounts,
+):
+    ## parameters
+    agent = contracts.agent
+    nor = contracts.node_operators_registry
+    staking_router = contracts.staking_router
+    target_NO_id = 1
+    target_validators_count_change_request = TargetValidatorsCountChanged(
+        nodeOperatorId=target_NO_id, targetValidatorsCount=0
+    )
```

Comment:
I'd moved these definitions upper (at the global scope level)

Reply:
it doesn't in global scope, probably there some initialisation between global scope execute and test call
![image](https://github.com/lidofinance/scripts/assets/3354239/73d0f45f-b30c-4ed4-8005-5a97bfc8efe5)